src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+@import "../fonts/fonts";
+@import "./blocks/nav";
+@import "./blocks/card";
+@import "./blocks/stars";
+@import "./blocks/header";
+@import "./blocks/container";
+
+:root {
+  --hover-color: #00acdc;
+  --card-width: 200px;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto-Medium", sans-serif;
+  --hover-color: #00acdc;
+  --card-width: 200px;
+}
+
+ul {
+  padding-left: 0;
+  margin: 0;
+}
+
+.logo {
+  position: relative;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+}
+
+.is-active {
+  color: var(--hover-color);
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  height: 4px;
+  width: 37px;
+  margin-top: 18.5px;
+  background-color: var(--hover-color);
+  border-radius: 8px;
+
+}
+
+.container_item {
```

Comment:
Move .container_item styles to the container block file, for better organization and readability.